### PR TITLE
Fix native-cql branch: $in with multiple same values

### PR DESCRIPTION
**Pull request by Yuqi-Du**
Bug in native-cql branch : if have multiple same values in the $in value array, json api will error out.
This pr is for the fix.
**Which issue(s) this PR fixes**:
Fixes https://github.com/stargate/jsonapi/issues/623

**Checklist**
- [x] Changes manually tested
- [x] Automated Tests added/updated
- [ ] Documentation added/updated
- [x] CLA Signed: [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

File: src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/ExpressionBuilder.java [hunk 1]
```diff
@@ -135,4 +135,31 @@ private static Expression<BuiltCondition> buildExpressionRecursive(
     return ExpressionUtils.buildExpression(
         conditionExpressions, logicalExpression.getLogicalRelation().getOperator());
   }
+
+  public static List<Object> getExpressionValuesInOrder(Expression<BuiltCondition> expression) {
```
Comment (tatu-at-datastax) [inline src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/ExpressionBuilder.java:139]: Could you add a Javadoc explaining what this does?

**Review by tatu-at-datastax — APPROVED** [review]
LGTM -- ideally would add Javadoc to explain intended behavior (although name is good and gives some idea).

**Comment by tatu-at-datastax** [comment]
@Yuqi-Du Is this not needed for `main`?

File: src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/ExpressionBuilder.java (hunk 1 above)
Comment (Yuqi-Du) [reply src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/ExpressionBuilder.java:139]: sure!

**Comment by Yuqi-Du** [comment]
> @Yuqi-Du Is this not needed for `main`?

Yes, only for CQL branch.

**Comment by tatu-at-datastax** [comment]
@Yuqi-Du may want to merge/rebase from `native_cql_changes` to get Mahesh' IT fixes, to let CI verify there are only those 4 Authorization fails and nothing else

**Pull request merged**